btr-web/btr-main-app/enums/percentage-range-e.ts

```diff
@@ -0,0 +1,7 @@
+export enum PercentageRangeE {
+  NO_SELECTION = 'noSelection',
+  LESS_THAN_25 = 'lessThan25',
+  BETWEEN_25_AND_50 = 'between25And50',
```

Comment:
can you change this to be 'atleast25To50' so that we know by reading it that it includes the 25? Similarly I think 'moreThan50To75' makes more sense since that range includes 75